robottelo/common/constants.py

```diff
@@ -144,6 +144,7 @@
 REPOSET = {
     'rhct6': "Red Hat CloudForms Tools for RHEL 6 (RPMs)",
     'rhel6': "Red Hat Enterprise Linux 6 Server (RPMs)",
+    'rhelc6': "Red Hat Enterprise Linux 6 Server - RH Common (RPMs)",
```

Comment:
never mind, all other repos are using this same format, we need to improve this later.